packages/components/util/types.ts

```diff
@@ -0,0 +1,50 @@
+// For now - no hardhat
+// import {
+//   ProviderConnectInfo,
+//   ProviderMessage,
+//   ProviderRpcError,
+//   RequestArguments,
+// } from 'hardhat/types';
+
+export interface EthereumEvent {
```

Comment:
could put this (and other type definitions we write that arent internal types) in the packages/@types folder